edelphi/src/main/java/fi/metatavu/edelphi/pages/panel/admin/report/thesis/ThesisMultipleScale2DQueryReportPage.java

```diff
@@ -0,0 +1,105 @@
+package fi.metatavu.edelphi.pages.panel.admin.report.thesis;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.eclipse.birt.chart.model.Chart;
+
+import fi.metatavu.edelphi.dao.querydata.QueryQuestionCommentDAO;
+import fi.metatavu.edelphi.domainmodel.panels.PanelStamp;
+import fi.metatavu.edelphi.domainmodel.querydata.QueryQuestionComment;
+import fi.metatavu.edelphi.domainmodel.querylayout.QueryPage;
+import fi.metatavu.edelphi.domainmodel.querylayout.QueryPageType;
+import fi.metatavu.edelphi.pages.panel.admin.report.util.ChartContext;
+import fi.metatavu.edelphi.pages.panel.admin.report.util.QueryReportPage;
+import fi.metatavu.edelphi.pages.panel.admin.report.util.QueryReportPageData;
+import fi.metatavu.edelphi.pages.panel.admin.report.util.ReportContext;
+import fi.metatavu.edelphi.smvcj.controllers.RequestContext;
+import fi.metatavu.edelphi.utils.QueryPageUtils;
+import fi.metatavu.edelphi.utils.QueryUtils;
+import fi.metatavu.edelphi.utils.ReportUtils;
+import fi.metatavu.edelphi.utils.RequestUtils;
+
+public class ThesisMultipleScale2DQueryReportPage extends AbstractThesisScale2DQueryReportPage {
+  
+  private static final Logger logger = Logger.getLogger(ThesisMultipleScale2DQueryReportPage.class.getName());
+  
+  private static final String CHART_INDEX_PARAM = "chartIndex";
+
+  public ThesisMultipleScale2DQueryReportPage() {
+    super(QueryPageType.THESIS_MULTIPLE_2D_SCALES);
+  }
+
+  @Override
+  public QueryReportPageData loadPageData(RequestContext requestContext, ReportContext reportContext, QueryPage queryPage) {
+    List<String> theses = QueryPageUtils.getListSetting(queryPage, "multiple2dscales.theses");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "multiple2dscales.theses" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)